bites_test.py

```diff
@@ -170,6 +165,28 @@ def test_adjacent_food_is_blocked_by_presence_of_other_ant(self):
     self.assertEqual(actual_food, expected_food)
     self.assertEqual(actual_new_trail, expected_new_trail)
 
+  def test_raises_value_error_if_direction_is_wrong(self):
+  # test 18
+    trail = ["apple", "grapes", "cheese", "bread"]
+    ant_positions = {"purple": 1, "yellow": 2}
+    ant = "purple"
+    direction = "forwards"
+    self.assertRaises(ValueError, take_food, trail, ant_positions, ant, direction)
+
+class InitialiseAnthillTest(unittest.TestCase):
+  def test_can_initialise_anthill_as_list_with_len_five_and_every_element_is_None(self):
+  # test 27  
+    expected_anthill = [None, None, None, None, None]
+    actual_anthill = initialise_anthill(ANTS)
```

Comment:
It would be preferable for this test not to depend on the ANTS constant, i. e. define `ants = [...]` and then call `initialise_antil(ants)` with this local variable instead of the ANTS constant.